autopush-common/src/db/bigtable/bigtable_client/mod.rs

```diff
@@ -1337,6 +1325,17 @@ mod tests {
             client.get_user(&uaid).await?.unwrap().connected_at
         );
 
+        // can we increment the storage for the user?
+        client
```

Comment:
Added test to verify we can increment storage (generally checks to see that we're not leaking a qualifier as the family id)